piwik-sdk/src/main/java/org/piwik/sdk/plugins/CustomDimensions.java

```diff
@@ -0,0 +1,22 @@
+package org.piwik.sdk.plugins;
+
+import org.piwik.sdk.TrackMe;
+
+/**
+ * This plugins allows you to track any Custom Dimensions defined in
+ * https://plugins.piwik.org/CustomDimensions
+ */
+public class CustomDimensions extends TrackMe {
+
+    /**
+     * This method sets a tracking API parameter dimension%dimensionId%=%dimensionValue%.
+     * Eg dimension1=foo or dimension2=bar.
+     * So the tracking API parameter starts with dimension followed by the set dimensionId.
+     * @return instance of CustomDimensions plugin
+     */
+    public synchronized CustomDimensions set(int dimensionId, String dimensionValue) {
```

Comment:
We should probably document the limitations here. See: https://github.com/piwik/plugin-CustomDimensions/issues/6

Actually we could enforce them.
* Check that the ```dimensionId```is valid. I guess it's 0-* or 1-*?
* ```dimensionValue``` is max 255 chars.

Sp if the paramaters are bad we could throw a warning and just not set them.